sdks/python/apache_beam/runners/interactive/interactive_runner.py

```diff
@@ -152,7 +152,11 @@ def run_pipeline(self, pipeline, options):
               user_pipeline)):
         streaming_cache_manager = ie.current_env().get_cache_manager(
             user_pipeline)
-        if streaming_cache_manager:
+
+        # Only make the server if it doens't exist already.
+        if (streaming_cache_manager and
+            not ie.current_env().get_test_stream_service_controller(
```

Comment:
Just wondering, what if the service controller has stopped? Is it possible?

Reply:
No, the server is "one-time" use. Once it stops it can't be started up again and this should be set None.